### PR TITLE
OOP - Milestone: instructions for players

### PR DESCRIPTION
**Pull request by julian3493**
### Instructions in the README file

- We write detailed instructions for each step into the game.
- We give instructions to start the game
- We explain how the Tic Tac Toe game works

### CODE REVIEW

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team    
#### Great Job so far, everything works fine, but there are a few more things required. 
### General 
- [x] Let's write detailed steps on how to clone and run the project locally 

### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team    
#### Great Job so far, everything works fine, but there are a few more things required. 
### General 
- [x] Let's write detailed steps on how to clone and run the project locally, for example

````ruby
Clone the repository to your local machine
$ git clone https://github.com/kobiyoyo/Tic-Tac-Toe.git
Go to the directory

$ cd tic-tac-toe
Go to the execution file directory

$ cd bin
Run the file

$ ruby main.rb
````

### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
...
+
+### Prerequisites
+
+- Computer
+- Terminal -- any terminal will work just fine.
+- Ruby Installed -- version 2.4.X and above
+
+### Setup
+
+- Open your terminal and go to the directory where you want to clone the repo.
```
Comment (euqueme) [inline README.md:23]: ### [OPTIONAL]

- [ ] Users don't necessarily need to clone the repo they can just download it too, let's mention this somewhere

**Review by euqueme — APPROVED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @julian3493 and @mke2111 :lollipop: :balloon: 

### Thank you for working on the changes requested by the previous reviewer, :medal_sports: :medal_sports:  your Readme file looks way better and your game works just fine :ok_hand: it's a yes for me Congratulations merge this branch and keep going :tada: :tada: :tada: 

### Please :pray: keep up the good work :clinking_glasses: 

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Pull request merged**